.goreleaser.yml

```diff
@@ -52,6 +52,34 @@ before:
         ".releaseSeries |= (. + [{contract: \"v1beta1\", major: {{ .Major }}, minor: {{ .Minor }}}] | unique)" \
         metadata.yaml >release-metadata.yaml'
     - make template-helm-repository
+    # dev image
+    - |
+      sh -ec 'if [ {{ .IsSnapshot }} == true ] ; then
```

Comment:
Its probably possible to combine these 2 `ko` steps and based on the `.IsSnapshot` condition build either a single platform or multiple, but wanted to keep the scope of these changes small.